src/components/controls/keyboard-controls.js

```diff
@@ -0,0 +1,94 @@
+var registerControls = require('../../core/controls').registerControls;
```

Comment:
I wonder if hotspot needs to be a control rather than a standard component? All it needs to do is tween the camera (wrapper) to the position of the hotspot. If the camera "collides" with something on the way (which it shouldn't with well-placed hotspots), then you just let it clip.